apps/zui/src/ppl/detail/util/formatDur.ts

```diff
@@ -14,7 +14,11 @@ export default function formatDir(start, end) {
     parts.push(formatPart(obj[unit], unit))
   }
   if (allZero) {
-    parts.push(formatPart((end.getTime() - start.getTime()) / 1000, "seconds"))
+    if (end.getTime() === start.getTime()) {
```

Comment:
Check if the two dates are the same to determine if the duration is less than 1ms.